README.md

```diff
@@ -15,34 +21,33 @@ Updated bundle for the browser is always attached to the GitHub Release.
 > **Note**
 > Read the [migration guide from v2 to v3](./docs/migrations/v2-to-v3.md).
 
-
 <!-- toc is generated with GitHub Actions do not remove toc markers -->
 
 <!-- toc -->
 
 - [Installation](#installation)
 - [Usage](#usage)
 - [Examples](#examples)
-  * [Example with parsing](#example-with-parsing)
```

Comment:
The TOC content is autogenerated, so please revert them back to previous values.